NEWS.md

```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.9.9] – 2023-12-14
+
+### Fixed
+
+* introduced a nonzero `atol` for all point and vector checks that compre to zero.
+  This makes those checks a bit more relaxed by default and resolves [#630](https://github.com/JuliaManifolds/Manifolds.jl/issues/630).
+* `get_estimation_method(M)` is deprecated, use `get_approximation_method(M, f)` for your specific mathod `f` on the manifold `M`.
```

Comment:
```suggestion
* `default_estimation_method(M, f)` is deprecated, use `default_approximation_method(M, f)` for your specific method `f` on the manifold `M`.
* `AbstractEstimationMethod` is deprecated, use `AbstractApproximationMethod` instead.
```
There was no `get_estimation_method`. `AbstractEstimationMethod` is deprecated though.

Reply:
There was in just a few places.

Reply:
Hm, I couldn't find it then :sweat_smile:

Reply:
See https://github.com/JuliaManifolds/Manifolds.jl/blob/b2d83b337e214158f64d22a39776b9a80ffd2981/src/manifolds/Grassmann.jl#L174-L176 (that's on current master) – but I also only saw them when reworking Manifolds.jl

Reply:
Yes, that's `default_estimation_method`, not `get_estimation_method`. I'm not sure where I was wrong?

Reply:
ah I just (as for the other comment) did not read your comment carefully enough, sorry. This PR was three days (not straight in a row) of fighting wrong dispatches. You are right, the fix is ok.

Reply:
OK, it's fine :slightly_smiling_face: . That was quite a lot of fighting then.